### PR TITLE
Maintainer Guide

### PR DESCRIPTION
**Pull request by it33**
Drafting maintenance info page to serve as a template for other open source projects to recognize maintainers

### CODE REVIEW

**Comment by it33** [comment]
Hi @xcompass, 

I started drafting this document to make it easier to update this repo when Mattermost issues a security update, but then it got longer and longer and now I have something that can potentially be re-used in other open source projects to help guide maintainers. 

Also, I was thinking it's important to recognize the contributions of maintainers and repo creators. 

Wondering if you had any feedback, or if this looked okay to merge?

File: MAINTENANCE.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Maintainer(s) should periodically review pull requests and issues submitted to provide feedback and to merge pull request changes when the maintainer feels the change would be appropriate. 
+
+## Credits 
+
+PREVIOUS MAINTAINERS 
+
+| Maintainer             | Start Date    | End Date   |
+|:-----------------------|:--------------|:-----------|
+| Yi EungJun - @npcode   | 2015-11-30    | 2015-11-26 |
```
Comment (xcompass) [inline MAINTENANCE.md]: Got the date backwards :)

File: MAINTENANCE.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Maintainer(s)
+
+The following people help to maintain this open source project: 
+
+| Current Maintainer(s)  | Start Date    | 
+|:-----------------------|:--------------|
+| Pan Luo - @xcompass    | 2015-11-30    |
+
+In case something happens where a maintainer is unable to complete their responsibilies or find a new maintainer, the following sponsoring organization can help find a new maintainer:
```
Comment (xcompass) [inline MAINTENANCE.md]: a maintainer is unable to complete his/her responsibilities?

File: MAINTENANCE.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+## Activities
+
+The following is a guide for current, new maintainers and prospective maintainers of this open source project to get started and to understand on-going responsibilities: 
+
+### Getting Started 
+
+The following steps should be completed by a new maintainer 
+
+1. **Add your name** - Create a pull request to add your name, GitHub username and start date to this document. 
+2. **Subscribe to mailings** - To be notified of new releases and security updates of Mattermost, subscribe to the [Mattermost Security Update Mailing List](http://mattermost.us11.list-manage.com/subscribe?u=6cdba22349ae374e188e7ab8e&id=3a93eb6929) and the [Mattermost Insiders Newsletter](http://mattermost.us11.list-manage.com/subscribe?u=6cdba22349ae374e188e7ab8e&id=2add1c8034)
```
Comment (xcompass) [inline MAINTENANCE.md]: Subscribe to mailing lists?

File: MAINTENANCE.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+### Updating 
+
+When receive a mailing list email about a new security update or major version of Mattermost being released, update the version number of this project by doing the following: 
+
+- Change the [version number](https://github.com/mattermost/mattermost-docker/blob/master/app/Dockerfile#L6) in the **master branch** to pull in the latest Mattermost Team Edition release
+- Change the [version number](https://github.com/mattermost/mattermost-docker/blob/team-and-enterprise/app/Dockerfile#L6) in the **the team-and-enterprise branch** to pull in the latest Mattermost Enterprise Edition release
+
```
Comment (xcompass) [inline MAINTENANCE.md]: add: - Tag the new version

**Comment by it33** [comment]
Thanks @xcompass 

- Propose keeping "their" over his/her as that seems to be the more modern way to note it. Opened an issue to figure this out, but thinking we just go with "their" for now? https://github.com/mattermost/docs/issues/842

- Other changes made

File: MAINTENANCE.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+## Maintainer(s)
+
+The following people help to maintain this open source project: 
+
+| Current Maintainer(s)  | Start Date    | 
+|:-----------------------|:--------------|
+| Pan Luo - @xcompass    | Nov 30 2015   |
+
+In case something happens where a maintainer is unable to complete their responsibilies or find a new maintainer, the following sponsoring organization can help find a new maintainer:
```
Comment (xcompass) [inline MAINTENANCE.md]: If you want keep "their", then you may want to change "where a maintainer is unable" to "where maintainers are unable" to be consistent with "their"

**Comment by it33** [comment]
Good point, thanks @xcompass!

**Pull request merged**